pennylane/return_types.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Class and functions for activating, deactivating and checking the new return types system
+"""
+# pylint: disable=too-few-public-methods
+
+
+class ReturnType:
+    """Class to store the attribute `activated` which indicates if the new return type system is on. Default=False."""
+
+    activated = False
```

Comment:
My preference was to avoid having `global` statement to change the variable. But I guess it is a question of `taste`. Anything you would suggest here?

Reply:
The module scope already makes it non-global. I guess it's just a matter of preference here, but I see no reason of adding an extra class for this.

If we're concerned about the user modifying it themselves, we can change the name to `__activated` and put it top-level, and it will automatically be hidden.